docs/guides/integration-javascript-client-side.md

```diff
@@ -281,7 +281,19 @@ In some cases, the user's DII is not available on page load, and getting the DII
 You can potentially avoid that cost by checking for an existing token that you can use or refresh. To do this, call
 [__uid2.isLoginRequired](../sdks/sdk-ref-javascript#isloginrequired-boolean) which returns a Boolean value. If it returns `true`, this means that the UID2 SDK cannot create a new advertising token with the existing resource and DII is required to generate a brand new UID2 token.
 
-The following code snippet demonstrates how you might integrate with the UID2 SDK for JavaScript for the two scenarios above&#8212;starting with no token as well as reusing/refreshing any existing UID2 token if found. 
+The only exception is if the following method returns true, 
+
+```js
+__uid2.hasOptedOut()
+```
+
+It indicates that the DII has been opted out of UID2 and no identity/token should be generated for it. While the UID2 SDK for JavaScript will respect the opt out preference and not generate UID2 tokens even if you call any of the `setIdentity` method calls with DII again, optionally, you might want to avoid making such calls repeatedly.
+
+The following code snippet demonstrates how you might integrate with the UID2 SDK for JavaScript for the three scenarios above:
+
+1. starting with no token 
+2. reusing/refreshing any existing UID2 token if found
+3. detect if the DII has opted out of UID2 and no token will ever be generated for it
```

Comment:
I think this phrasing is a little confusing - I think the original wording was clearer. I don't think we need to tell them that we're also handling the optout scenario here.

Reply:
ok reverted back to the original wording